config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -55,21 +55,23 @@ presubmits:
           - ./scripts/ci-e2e.sh
         env:
           - name: GINKGO_FOCUS
-            value: "Workload cluster creation"
+            value: \[REQUIRED\]
           - name: GINKGO_SKIP
-            value: "Creating a GPU-enabled cluster|.*Windows.*dockershim.*|.*AKS.*|Creating a cluster that uses the external cloud provider"
+            value: ""
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: capz-pr-e2e-main
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
-  - name: pull-cluster-api-provider-azure-e2e-full
+  - name: pull-cluster-api-provider-azure-e2e-flavors-optional
```

Comment:
@CecileRobertMichon @shysank how do we feel about:

1) renaming the "-full" test job to "-flavors-optional" for PR tests?
2) running only the set of cluster tests that the "-e2e" job doesn't run

Reply:
> 2. running only the set of cluster tests that the "-e2e" job doesn't run

👍 

> 1. renaming the "-full" test job to "-flavors-optional" for PR tests?

Slight preference for naming this one "pull-cluster-api-provider-azure-e2e-optional" to be consistent with "-e2e" (default) and "-e2e-exp" (exp flavors)

Reply:
let's also update the PR description to call attention to this change

Reply:
done